tark/tark/templates/privacy_notice_tark.html

```diff
@@ -74,6 +74,13 @@ <h3>6. How long do we keep your personal data?</h3>
                 <li> We will anonymise our web logs by removing the recorded IP address after 30 days. We will retain
                     relevant security logs for 90 days.
                 </li>
+                <li>
```

Comment:
The privacy notice: http://tark.ensembl.org/privacy_notice_tark isn't linked anywhere in the app.

The one at the top bar takes the user to the EBI website https://www.ebi.ac.uk/data-protection/privacy-notice/transcript-archive-tark- where the paragraph added here doesn't exist.